### DISCUSSION TITLE
Should the `.serverless_nextjs` directory be cached like `.serverless`?

### DISCUSSION THREAD

**Discussion by 3stacks** [serverless-nextjs/serverless-next.js#2159] [Q&A] [answered]
As per title... I'm currently building in Github Actions and before deploying I download the `.serverless` directory from s3. Should I be doing the same thing with `.serverless_nextjs`?

**Comment by dphang** [comment, marked as answer]
Hi there, the `.serverless_nextjs` directory does not need to be cached since it's the artifacts (lambda code, assets) does get regenerated/repackaged on each build from the sources it uses (`.next`)
